content/go/concepts/math-functions/terms/atanh/atanh.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'Atanh()'
+Description: 'Returns the inverse hyperbolic tangent of the given value.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Math'
+  - 'Trigonometry'
+  - 'Functions'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`Atanh()`** function takes a float64 argument representing the angle whose inverse hyperbolic tangent value is to be returned. It returns this inverse hyperbolic tangent as a float64 value that can range from negative infinity to positive infinity.
+
+# Syntax
+
+```pseudo
+result := math.Atanh(x)
+```
+
+Where `x` represents the angle whose inverse hyperbolic tangent value is to be returned.
+
+And where the result is the inverse hyperbolic tangent of `x`.
```

Comment:
Replace this line with "notable cases" that highlights results given specific values of x, e.g. result = -Inf when x = -1

Reply:
Hello, @caupolicandiaz 

Thanks for the feedback.

Could you please help me understand a little better what you're looking for with regards to the "notable cases" section?

From your description, I find it very similar to the currently documented "Example" section, and I'm not sure to understand the differences.

If you could elaborate, I would sure appreciate it. It'd help me do a better job at complementing the documentation.

Once again, thank you for your feedback!

Reply:
Take a look at this example
https://www.codecademy.com/resources/docs/go/math-functions/cosh

Reply:
Excuse the delay, @caupolicandiaz .

The notable cases of when 1 or -1 are passed to the function have been documented.